pyht/async_client.py

```diff
@@ -146,8 +146,8 @@ async def refresh_lease(self):
             if self._rpc is None:
                 insecure = self._advanced.insecure or "on-prem.play.ht" in grpc_addr
                 channel = (
-                    insecure_channel(grpc_addr) if insecure
-                    else secure_channel(grpc_addr, ssl_channel_credentials())
+                    insecure_channel(grpc_addr, options=CLIENT_RETRY_OPTIONS) if insecure
```

Comment:
Seems risky to make this the default without lots of testing.  It could change all sorts of behavior. I think you should offer this as an advanced option and tell the customer to set that option.  Would be nice if you did this through the existing congestion_ctrl mechanism (e.g. congestion_ctrl=CongestionCtrl.GRPC_RETRY_MAR_2024) - but not a huge deal if you do it some other way.